### ISSUE TITLE
My controller stops working after install the emulator

### ISSUE THREAD

**Issue by caosprestes** [starshinata/PS4-Keyboard-and-Mouse-Adapter#68]
### Have you read the FAQ ?
* [ x] I have read the FAQ which is available at https://github.com/starshinata/PS4-Keyboard-and-Mouse-Adapter/blob/master/documentation/frequently-asked-questions.md

### You are raising an Issue, what can you generalise it as ?
* [ x] Problem - something is wrong
* [ ] Enhancement suggestion - i have an idea to make something better
* [ ] Other

### What Console are you using
* [ ] PS5
* [ x] PS4

### What version of PS4KMA are you using ?
3.1.0

### What emulation method are you using ?
* [ ] ViGEm and NO process injection for bottom toolbar
* [ x] ViGEm and process injection for bottom toolbar *recommended*
* [ ] NO ViGEm and just process injection

### Describe your issue
My controller stops working after install the emulator, before it was working using USB connection. Now it's not working, I was testing keyboard and mouse setup for some games, but other I would like to play in the Controller, but now Remote Play seems to not recognize it anymore. I can connect using Bluetooh, but Remote Play only accept UBS input.

### If problem, how can someone else replicate this?

Use UBS Input DS4, install the apps, try using again.

**Comment by pancakeslp** [comment]
Stupid question, I assume you have turned off ps4kma and closed the remote
play window, then started a fresh remote play window before trying to use
your controller with remote play. Please confirm ?

**Comment by caosprestes** [comment]
Yes, I closed and uninstall all related apps, the controller kept not working. Now to solve the problem immediately I just recover my system before the installations and the controller is working again.

**Comment by pancakeslp** [comment]
I was away at the time, and I'm back at home now and I cant replicate.
I have latest PS4kma installed and and vigem drivers installed and can use my ps4 controller on remote play.

I suspect maybe your ps4 controller was still connected to the computer via bluetooth, or connected to the PS4.
If you can think of anything else that might be pertinent please let me know.
